pointer.go

```diff
@@ -104,6 +116,12 @@ func (p Pointer) Descendant(path string) (Pointer, error) {
 	return append(p, dpath...), nil
 }
 
+// RawDescendant extends the pointer with 1 or more path tokens
+// The function itself is unsafe but allows for much faster pointer management
+func (p Pointer) RawDescendant(path ...string) (Pointer, error) {
+	return append(p, path...), nil
+}
+
```

Comment:
This isn't unsafe 😄! If the concern is potentially exceeding the cap, the go runtime will just re-allocate to a new pointer once the cap is exceeded:

https://stackoverflow.com/a/41668362

To that end, what about just calling this `Descendant`, or `NewDescendant`, and dropping the error?

Reply:
Ah, the "unsafe" part was more aimed at we don't completely re-parse the pointer and deal with the ecape/unescape dance.